core/conn/jdbc_type2/native/Debug.cpp

```diff
@@ -183,8 +184,9 @@ static const char *UCase(const char *text)
 
 static unsigned long GetThreadId(void)
 {
-	_TSLX_t thread_id = tslx_ext_pthread_self();
-	return((unsigned long) thread_id.field1);
+	//_TSLX_t thread_id = tslx_ext_pthread_self();
+	//return((unsigned long) thread_id.field1);
+        return pthread_self();
```

Comment:
It should be ok to use the same JIRA to remove the unwanted files. If you prefer a different JIRA,  can you please submit a JIRA yourself using https://issues.apache.org/jira/browse/TRAFODION/
It is difficult to assign the JIRA to you if I file it.